packages/dashboard/src/customization/propertiesSections/textSettings/text.tsx

```diff
@@ -85,8 +85,8 @@ const TextSettings: FC<TextSettingsProps> = ({
   return (
     <ExpandableSection headerText={defaultMessages.title} defaultExpanded>
       <div className='text-configuration' style={{ gap: awsui.spaceScaledS }}>
-        <label>{defaultMessages.color}</label>
-        <ColorPicker color={fontColor} updateColor={updateFontColor} />
+        <label htmlFor='color-picker'>{defaultMessages.color}</label>
+        <ColorPicker id='color-picker' color={fontColor} updateColor={updateFontColor} />
```

Comment:
Can we make this more specific for text settings color picker? Also, do our other color pickers in the properties / alarms section need this also?

Reply:
I have changed the id to be specific to 'text' now. Also this change was requested for configPanelText colorPicker label only.